kedro/template/{{ cookiecutter.repo_name }}/kedro_cli.py

```diff
@@ -121,8 +121,13 @@ def test(args):
 
 
 @cli.command()
+@click.option("--conda")
 def install():
-    """Install project dependencies from requirements.txt."""
+    """Install project dependencies from both requirements.txt and environment.yml (optional)."""
+
+    if conda:
+        call("conda", ["install", "--file src/environment.yml", "--yes"])
```

Comment:
This won't work, unless it looks something like this:

```suggestion
        call(["conda", "install", "--file", "src/environment.yml", "--yes"])
```